src/ExportMap.js

```diff
@@ -379,6 +379,20 @@ ExportMap.parse = function (path, content, context) {
 
   function captureDependency(declaration) {
     if (declaration.source == null) return null
+    const importedSpecifiers = new Set()
+    if (declaration.specifiers) {
+      declaration.specifiers.forEach(specifier => {
+        if (specifier.type === 'ImportDefaultSpecifier') {
```

Comment:
We could probably shorten this a bit with something like:
```suggestion
        const supportedTypes = new Set(['ImportDefaultSpecifier', 'ImportNamespaceSpecifier', 'ImportSpecifier']);
        if (supportedTypes.has(specifier.type)) {
          importedSpecifiers.add(specifier.type);
        }
```